tsconfig.json

```diff
@@ -1,18 +1,24 @@
 {
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "_version": "20.1.0",
```

Comment:
Curious - is this required? I think VSCode (and other IDEs) recognise this file schema by default

Reply:
You're right, it seems to not make any difference. I'll delete it.